### PR TITLE
Rename the go packages to match the repo name and URL

### PR DESCRIPTION
**Pull request by aqrln**
https://github.com/prisma/prisma-engines/pull/4357 will be obsolete after this as it also automatically makes the binary name `planetscale-proxy`.

### CODE REVIEW

File: types/psdb/v1alpha1/database.pb.go [hunk 1]
```diff
@@ -343,8 +343,8 @@ var file_psdb_v1alpha1_database_proto_rawDesc = []byte{
 	0x73, 0x74, 0x1a, 0x1e, 0x2e, 0x70, 0x73, 0x64, 0x62, 0x2e, 0x76, 0x31, 0x61, 0x6c, 0x70, 0x68,
 	0x61, 0x31, 0x2e, 0x45, 0x78, 0x65, 0x63, 0x75, 0x74, 0x65, 0x52, 0x65, 0x73, 0x70, 0x6f, 0x6e,
 	0x73, 0x65, 0x22, 0x00, 0x42, 0x46, 0x5a, 0x44, 0x67, 0x69, 0x74, 0x68, 0x75, 0x62, 0x2e, 0x63,
-	0x6f, 0x6d, 0x2f, 0x6d, 0x61, 0x74, 0x74, 0x72, 0x6f, 0x62, 0x65, 0x6e, 0x6f, 0x6c, 0x74, 0x2f,
-	0x70, 0x73, 0x2d, 0x68, 0x74, 0x74, 0x70, 0x2d, 0x73, 0x69, 0x6d, 0x2f, 0x74, 0x79, 0x70, 0x65,
+	0x6f, 0x6d, 0x2f, 0x70, 0x72, 0x69, 0x73, 0x6d, 0x61, 0x2f, 0x70, 0x6c, 0x61, 0x6e, 0x65, 0x74,
+	0x73, 0x63, 0x61, 0x6c, 0x65, 0x2d, 0x70, 0x72, 0x6f, 0x78, 0x79, 0x2f, 0x74, 0x79, 0x70, 0x65,
```
Comment (aqrln) [inline types/psdb/v1alpha1/database.pb.go:347]: in ascii:

```diff
- om/mattrobenolt/ps-http-sim/type
+ om/prisma/planetscale-proxy/type
```

**Comment by Jolg42** [comment]
Why not keep the original name and keep the fork as a 100% copy of the original repo?

**Comment by aqrln** [comment]
@Jolg42 because I'm not very familiar with Go and wasn't sure if importing `github.com/mattrobenolt/ps-http-sim/types/psdb/v1alpha1` and `github.com/mattrobenolt/ps-http-sim/types/psdb/v1alpha1/psdbv1alpha1connect` in `main.go` would pull the upstream repo instead of using the local sources, and I needed to make sure the local sources were used for debugging. 

I now double checked that it actually doesn't, and it's fine to leave the original module names, so we can drop this PR.

**Comment by Jolg42** [comment]
Thanks for checking this!
Then I'll close this now as you mentioned that we can drop it.

**Pull request closed without merging**